src/Component/TaskItems/TaskItems.js

```diff
@@ -1,37 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import TaskItem from './TaskItem';
+import TaskItemEditModal from './TaskItemEditModal';
 
 const TaskItems = () => {
+
+    // const [object, seObject] = useState({});
+    const [name, setName] = useState('Walking');
+    const [asignee, setAsignee] = useState('Shaon');
+    const [label, setLabel] = useState('Beginner');
+    const [status, setStatus] = useState('Completed');
+    const [modalPreview, setmodalPreview] = useState(false);
+
     const TaskItems = [
         {
-            'name': 'Walking',
```

Comment:
Why remove this items?

Reply:
Couldn't work with those data. Need discussion vai.

Reply:
Let's sit and discuss